src/components/Display/Display.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './Display.scss';
+
+const Display = ({ displayValue }) => (
+    <div></div>
+)
+
+Display.propTypes = { displayValue: PropTypes.string.isRequired }
```

Comment:
```suggestion
Display.propTypes = { displayValue: string.isRequired }
```
From import update suggestion earlier in this file.